### PR TITLE
doc: Add tenant and cluster selector to the Server Addresses page

### PR DESCRIPTION
**Pull request by pradeeka7**
#### Summary

Refer to the issue 'Improve the page 'Server Addresses' to easily find the server addresses' https://github.com/TheThingsIndustries/lorawan-stack-docs/issues/1147

#### Screenshots

image

#### Changes

- Added tenant id and cluster id selector to dynamically change the server addresses.
- ...

#### Notes for Reviewers

@nejraselimovic 

#### Checklist

- [X] Scope: The referenced issue is addressed, there are no unrelated changes.
- [X] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [X] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [X] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Review by KrishnaIyer — COMMENTED** [review]
@pradeeka7: Do you have a screenshot on how the sections with the links to this selector work?

**Comment by pradeeka7** [comment]
localhost_1313_the-things-stack_concepts_server-addresses_(Laptop with HiDPI screen) (1)

localhost_1313_the-things-stack_concepts_server-addresses_(Laptop with HiDPI screen)

localhost_1313_the-things-stack_concepts_server-addresses_(Laptop with HiDPI screen) (2)

**Comment by pradeeka7** [comment]
Do you feel this sentence is weird after updating with the tenant ID?
image

**Comment by pradeeka7** [comment]
> completely removed because it's clear anyway

Agree. I think removing is better.

File: doc/content/the-things-stack/concepts/server-addresses/_index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Deployments of {{% tts %}} in a single cluster use the same server address for a
 Distributed deployments, like {{% tts %}} Cloud and Community Edition, use different addresses per region for routing components, while the address of global components (like Identity Server) is always the same. Enterprise and Open Source deployments may be in a single cluster, or distributed over multiple clusters with multiple addresses.
 The server addresses for the different {{% tts %}} deployments are listed below.
 
-You can update the examples below with your tenant ID and cluster ID by filling them here.
+You can update the examples below with your tenant ID and cluster ID by filling them here.  The `<tenant-id>` will be replaced with the provoded tenant ID.
```
Comment (nejraselimovic) [inline doc/content/the-things-stack/concepts/server-addresses/_index.md:17]:
```suggestion
You can update the examples below with your tenant ID and cluster ID by filling them here.  The `<tenant-id>` will be replaced with the provided tenant ID.
```

File: doc/content/the-things-stack/concepts/server-addresses/_index.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ The LNS gateway address is a combination of the **protocol** (wss), the **server
 
 **Example 1**: The <code data-content="cluster-address"><span data-content="cluster-id"></span></code> Community Edition LNS address is <code data-content="cluster-address">wss://<span data-content="cluster-id"></span>.cloud.thethings.network:8887</code>.
 
-**Example 2**: The LNS address for an <code data-content="cluster-address"><span data-content="cluster-id"></span></code> Cloud tenant is <code data-content="cluster-address">wss://<span data-content="tenant-id"></span>.<span data-content="cluster-id"></span>.cloud.thethings.industries:8887</code>, where `tenant` should be replaced with your Tenant ID.
+**Example 2**: The LNS address for an <code data-content="cluster-address"><span data-content="cluster-id"></span></code> Cloud tenant is <code data-content="cluster-address">wss://<span data-content="tenant-id"></span>.<span data-content="cluster-id"></span>.cloud.thethings.industries:8887</code>.
```
Comment (nejraselimovic) [inline doc/content/the-things-stack/concepts/server-addresses/_index.md:47]:
```suggestion
**Example 2**: The LNS address for a <code data-content="cluster-address"><span data-content="cluster-id"></span></code> Cloud tenant is <code data-content="cluster-address">wss://<span data-content="tenant-id"></span>.<span data-content="cluster-id"></span>.cloud.thethings.industries:8887</code>.
```

File: doc/content/the-things-stack/concepts/server-addresses/_index.md [hunk 3]
```diff
@@ -54,7 +54,7 @@ The CUPS gateway address is a combination of the **protocol** (https), the **ser
 
 **Example 1**: The <code data-content="cluster-address"><span data-content="cluster-id"></span></code> Community Edition CUPS address is <code data-content="cluster-address">https://<span data-content="cluster-id"></span>.cloud.thethings.network:443</code>.
 
-**Example 2**: The CUPS address for an <code data-content="cluster-address"><span data-content="cluster-id"></span></code> Cloud tenant is <code data-content="cluster-address">https://<span data-content="tenant-id"></span>.<span data-content="cluster-id"></span>.cloud.thethings.industries:443</code>, where `tenant` should be replaced with your Tenant ID.
+**Example 2**: The CUPS address for an <code data-content="cluster-address"><span data-content="cluster-id"></span></code> Cloud tenant is <code data-content="cluster-address">https://<span data-content="tenant-id"></span>.<span data-content="cluster-id"></span>.cloud.thethings.industries:443</code>.
```
Comment (nejraselimovic) [inline doc/content/the-things-stack/concepts/server-addresses/_index.md:57]:
```suggestion
**Example 2**: The CUPS address for a <code data-content="cluster-address"><span data-content="cluster-id"></span></code> Cloud tenant is <code data-content="cluster-address">https://<span data-content="tenant-id"></span>.<span data-content="cluster-id"></span>.cloud.thethings.industries:443</code>.
```

**Review by nejraselimovic — CHANGES_REQUESTED** [review]

File: doc/content/the-things-stack/concepts/server-addresses/_index.md (hunk 3 above)
Comment (pradeeka7) [reply doc/content/the-things-stack/concepts/server-addresses/_index.md:57]: @nejraselimovic Thanks for catching up on the typo.

**Pull request merged**